source/process/sg_document-structure.rst

```diff
@@ -0,0 +1,30 @@
+==================
+Document structure
+==================
+
+Structure and organization are an important part of a document's ease of use and its understandability. Information should be organized and presented in a logical order, with similar subjects grouped together in the same section.
+
+In most cases, a document has a title, an introductory paragraph, and one or more sections.
+
+Try to keep only one topic in a page. Shorter topics are easier to reuse in other documents, are easier to write and edit, and are easier to translate.
+
+Document title
+==============
+
+Use a title that accurately reflects the content of the document. People scan the table of contents looking for answers; it's often faster than using the built-in search engine.
+
+Use title case for document titles. For more information and an example of capitalization, see :ref:`capital`.
+
+Introductory paragraph
+======================
+
+Each page should have an introduction that acts as a short description of the document. The short description should be a single paragraph of no more than 3 sentences. Keep in mind that the description is displayed in the search results along with the page title. People read the description to help them decide if the document is the one that they want.
+
+Document sections
+=================
+
+To make pages easier for people to quickly scan for the content that they're looking for, break your document up into logical sections. Each section should have a title, and the title should relate to the content of the section.
+
+A section title is not required if you have only one section.
+
+Avoid subsections. If you find that you need subsections, quite often the document is too long, or too complex, and needs to be broken up into several pages.
```

Comment:
There are two approaches to this. One is to have one large monolithic file. The other is to have a docset made up of standalone files that are used where needed via `:toctree:`. I suppose there are pros and cons to each approach, but I would advocate for more granular, standalone, topic-based files for all Mattermost docs, not just the style guide, for the following reasons:

1. As a general rule, the content is easier to manage when it's broken up into standalone files with one topic per file. For example, if the MySQL config is one standalone file, it can be pulled into any number of Administration doc sets using `:toctree:` in Sphinx. If something changes and the MySQL instructions need updating, then the change is made in one place and that change automatically propagates to every doc set. If instead the MySQL content is copy/pasted into a dozen long pages, then you need to make sure that you catch every occurrence.
2. Content is easier to find for end users. For example, search the docs for the term `database installation` (https://docs.mattermost.com/search.html?q=database+installation&check_keywords=yes&area=default) and try to figure out which one of the results has your answer. If instead there was a topic called "Installing PostgreSQL database" it would most likely show up in the search results. Furthermore, when you clicked that result, the page would open and the installation instructions would be right there. As it is, none of the search results look promising, so you end up scanning the toc and clicking around until you find something.
	
Having said that, it looks like a lot of open source products are going for the monolithic style these days. I suppose you can make use of the `:include:` directive, which would allow standalone files in the repo and a monolithic file appearing in the published docs. This would leave the options open.